### ISSUE TITLE
Exact versions of dependencies not specified

### ISSUE THREAD

**Issue by samslattery** [wavefunction91/GauXC#31]
Because exact versions of the dependencies are not specified, changes in those repositories may cause problems in GauXC. For example: recent updates to gau2grid seem to have caused `gau2grid.h` to not be found in GauXC.

**Comment by wavefunction91** [comment]
Good call, I'll add this once the dust settles on the `cmake` branch

**Comment by wavefunction91** [comment]
@samslattery Let me know if https://github.com/wavefunction91/GauXC/commit/4239fd450b654ef3ecd391790c140c8ab3ee5f22 will fix your problems (or more rather, the tail of that PR)

**Comment by samslattery** [comment]
@wavefunction91 Thank you for addressing this! After changing to this commit, I'm getting a CMake issue with calling export on one of my targets that depends on gauxc. The error message says gauxc is not in any export set. After adding the below to `src/CMakeLists.txt` I was able to compile:
```
export(EXPORT gauxc-targets
      NAMESPACE GauXC::
      FILE "${PROJECT_BINARY_DIR}/GauXCTargets.cmake")
```

**Comment by wavefunction91** [comment]
Both this and the original issue will be closed by #14
